.github/workflows/unit-tests.yaml

```diff
@@ -86,3 +86,16 @@ jobs:
 
     - name: UI Unit Tests
       run: make ui-test
+
+  shell:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.50
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Similar to https://github.com/stackrox/stackrox/pull/4127#discussion_r1053243794 , maybe just omit the `ref`?